CWE/events/Italy ahistorical.txt

```diff
@@ -2966,7 +3038,14 @@ country_event = {
 			relation = { who = THIS value = 100 }
 		}
 		
-		SLO = { 3345 = { secede_province = ITA change_province_name = "Capodistria" } } #Koper, Istria
+		#Koper, Istria
+		random_country = { 
+		    limit = { is_vassal = no war = no owns = 3345 } 
+			3345 = { secede_province = ITA change_province_name = "Capodistria" }
```

Comment:
Iirc there’s a hardcoded bug preventing this. Secede province cannot work if it is fired by a 3P. It must be done by the country that owns the province. So this has to be put in 8020453 instead